app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala

```diff
@@ -22,7 +22,15 @@ import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
 class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
-  val cookieBannerAssetsPrefix: String = config.getOptional[String]("cookie-banner.assets-prefix").getOrElse("")
+  private def loadConfig(key: String): String =
```

Comment:
I think we discussed that loadConfig could be be deleted in favour of servicesConfig.getString(